src/framework/core/js/FluidIoC.js

```diff
@@ -1499,51 +1806,154 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         return instantiator.parseEL(shadow.path);
     };
 
-    /** Construct a component with the supplied options at the specified path in the component tree. The parent path of the location must already be a component.
+    /** Begin a fresh transaction against the global component tree. Any further calls to `fluid.registerPotentia`,
+     * `fluid.construct` or `fluid.destroy` may be contextualised by this transaction, and then committed as a single
+     * unit via `fluid.commitPotentiae` or cancelled via `fluid.cancelTreeTransaction`.
+     * @param {Object} [transactionOptions] - [optional] A set of options configuring this tree transaction. This may include fields
+     *     {String} breakAt - one of the values:
```

Comment:
Is there some some reason you can only break at one of these 2 places, and not potentially both?  
`breakAt: ["shells", "observation"]`

Reply:
One of them happens before the other one. If you break at the first, you won't reach the other